solutions/security-enforced/DA-cbr_rules.md

```diff
@@ -0,0 +1,62 @@
+# Configuring complex inputs for Event Notifications in IBM Cloud projects
```

Comment:
No need to duplicate this, its already in fully-configurable -we will just point variable description to that one